kafka/client_async.py

```diff
@@ -28,7 +28,9 @@
 from .metrics.stats import Avg, Count, Rate
 from .metrics.stats.rate import TimeUnit
 from .protocol.metadata import MetadataRequest
-from .protocol.produce import ProduceRequest
+
```

Comment:
Can you remove this empty line?

Reply:
ha hey if more people nitpicked about PEP-8 stuff, the world would be a happier and safer place!

I am on it. One moment